jaxlib/lapack.pyx

```diff
@@ -1525,7 +1525,10 @@ cdef void _unpack_float_eigenvectors(
 cdef void lapack_sgeev(void* out_tuple, void** data) nogil:
   cdef int b = (<int32_t*>(data[0]))[0]
   cdef int n = (<int32_t*>(data[1]))[0]
-  cdef const float* a_in = <float*>(data[2])
+  cdef char jobvl = (<uint8_t*>(data[2]))[0]
```

Comment:
To make this change backward compatible, you'll have to test the version of jaxlib in the translation rule.

Reply:
Since the PR now has only the lapack.pyx change, I think that the change should be backwards compatible. 

Keep in mind also that the CI tests will not test this change, because they will use a pre-built jaxlib. You may want to build a jaxlib from source and make sure that the tests still work.

Reply:
To make sure everything was fine, I recompiled `jaxlib` again and ran all the tests.
There are some errors but they do not seem to come from this change (they also occur when recompiling an earlier version of `jaxlib` without the present changes):
```bash
============================== short test summary info ===============================
FAILED tests/masking_test.py::MaskingTest::test_conv_padding=SAME_lhs_dilation=None_dimension_numbers=('NCHW', 'HWIO', 'NHWC')_lhs_perm=(0, 1, 2, 3)_rhs_perm=(2, 3, 1, 0)_out_perm=(0, 2, 3, 1)
FAILED tests/masking_test.py::MaskingTest::test_conv_padding=SAME_lhs_dilation=None_dimension_numbers=('NCHW', 'OIHW', 'NCHW')_lhs_perm=(0, 1, 2, 3)_rhs_perm=(0, 1, 2, 3)_out_perm=(0, 1, 2, 3)
FAILED tests/masking_test.py::MaskingTest::test_conv_padding=SAME_lhs_dilation=None_dimension_numbers=('NHWC', 'HWIO', 'NHWC')_lhs_perm=(0, 2, 3, 1)_rhs_perm=(2, 3, 1, 0)_out_perm=(0, 2, 3, 1)
FAILED tests/masking_test.py::MaskingTest::test_conv_strided_padding=SAME_lhs_dilation=None_dimension_numbers=('NCHW', 'HWIO', 'NHWC')_lhs_perm=(0, 1, 2, 3)_rhs_perm=(2, 3, 1, 0)_out_perm=(0, 2, 3, 1)
FAILED tests/masking_test.py::MaskingTest::test_conv_strided_padding=SAME_lhs_dilation=None_dimension_numbers=('NCHW', 'OIHW', 'NCHW')_lhs_perm=(0, 1, 2, 3)_rhs_perm=(0, 1, 2, 3)_out_perm=(0, 1, 2, 3)
FAILED tests/masking_test.py::MaskingTest::test_conv_strided_padding=SAME_lhs_dilation=None_dimension_numbers=('NHWC', 'HWIO', 'NHWC')_lhs_perm=(0, 2, 3, 1)_rhs_perm=(2, 3, 1, 0)_out_perm=(0, 2, 3, 1)
```

This suggest that the present change works as intended, but maybe I should file an issue for the tests above?